src/components/helpers/position_hook.ts

```diff
@@ -44,16 +44,39 @@ export function useAbsolutePosition(ref: Ref): DOMCoordinates {
       return;
     }
     const { top, left } = el.getBoundingClientRect();
-    if (left !== position.x || top !== position.y) {
-      position.x = left;
-      position.y = top;
-    }
+    position.x = left;
+    position.y = top;
   }
   onMounted(updateElPosition);
   onPatched(updateElPosition);
   return position;
 }
 
+/**
+ * Return the component (or ref's component) BoundingRect, relative
+ * to the upper left corner of the screen (<body> element).
+ *
+ * Note: when used with a <Portal/> component, it will
+ * return the portal position, not the teleported position.
+ */
+export function useBoundingRect(ref: Ref): Rect {
```

Comment:
`useBoundingRect` is not very clear as function name, what do you think about `useAbsoluteBoudingRect` ?

Reply:
Having `useAbsolutePosition` return a Rect seems very counter-intuitive tho. We could drop `useAbsolutePosition` and replace it by `useAbsoluteBoudingRect `. It's just that the name is a bit less good.

And yeah I forgot the Image Menu was separated, my bad... I'll see if I can do a PR to unify them because that's a bit stupid.